config/.base_rubocop.yml

```diff
@@ -70,8 +70,13 @@ Style/RedundantSelf:
 #     f x
 #     f x
 #   }.compact
+# do .. end から更にメソッドチェーンすると見づらいので
+# auto-correct せず、自分で修正する
+# spec 内は見た目が綺麗になるので許可
```

Comment:
I think it should be written in English to let all developers know about this message because there are some developers who don't speak Japanese in our organization.

Reply:
OK